src/OrleansAzureUtils/Providers/Streams/AzureQueue/AzureQueueBatchContainer.cs

```diff
@@ -102,6 +112,14 @@ public bool ImportRequestContext()
             return false;
         }
 
+        public void GetObjectData(SerializationInfo info, StreamingContext context)
```

Comment:
The only place where <code>AzureQueueBatchContainer</code> is being deserialized is at [AzureQueueBatchContainer.cs#L87](https://github.com/dotnet/orleans/blob/fc0af9079413505de6c5dad928a266fccda74d68/src/OrleansAzureUtils/Providers/Streams/AzureQueue/AzureQueueBatchContainer.cs#L87) and the sequence token will be created with <code>sequenceId</code> as parameter.